charts/newrelic-logging/README.md

```diff
@@ -167,12 +167,33 @@ See [values.yaml](values.yaml) for the default values
 | `dnsConfig`                                                  | [DNS configuration](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config) that will be added to the pods. Can be configured also with `global.dnsConfig`.                                                                                                                                                                                         | `{}`                                                                            |
 | `fluentBit.criEnabled`                                       | We assume that `kubelet`directly communicates with the container engine using the [CRI](https://kubernetes.io/docs/concepts/overview/components/#container-runtime) specification. Set this to `false` if your K8s installation uses [dockershim](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/) instead, in order to get the logs properly parsed. | `true`                                                                          |
 
+### Fluent Bit persistence modes
+
+Fluent Bit uses a database file to keep track of tailed and sent log lines from files, this database file is by default stored on the node using a `hostPath` mount. It's specifically stored (by default) on `/var/log/flb_kube.db` to keep things simple, as we're already mounting `/var` for accessing container logs.
+
+Sometimes the security constraints of some clusters doesn't allow to mount hostPath in read-write mode, that's why you can chose among the following
+persistence modes. Each one have their pros and cons.
+
+- `hostPath` (default) will use hostPath mount to store the db file on the node disk, the easier an most reliable, but prohibited by some security policies.
+- `none` will disable the fluent-bit db file, this could cause log duplication or data loss in case fluent-bit gets restarted.
+- `persistentVolume` (linux only) will use a ReadWriteMany persistent volume to store the db file. This will override `fluentBit.db` path and use `/db/${NODE_NAME}-fb.db` file instead. If you use this option in a windows cluster it will default to none on windows nodes.
```

Comment:
```suggestion
- `persistentVolume` (Linux only) will use a `ReadWriteMany` persistent volume to store the DB file. This will override the `fluentBit.db` path and use `/db/${NODE_NAME}-fb.db` instead. If you use this option in a Windows cluster it will default to `none` on Windows nodes.
```